llama_stack/providers/inline/agents/meta_reference/agent_instance.py

```diff
@@ -244,22 +244,13 @@ async def create_and_execute_turn(self, request: AgentTurnCreateRequest) -> Asyn
             )
             await self.storage.add_turn_to_session(request.session_id, turn)
 
-            if output_message.tool_calls and request.allow_turn_resume:
-                chunk = AgentTurnResponseStreamChunk(
-                    event=AgentTurnResponseEvent(
-                        payload=AgentTurnResponseTurnAwaitingInputPayload(
-                            turn=turn,
-                        )
-                    )
-                )
-            else:
-                chunk = AgentTurnResponseStreamChunk(
-                    event=AgentTurnResponseEvent(
-                        payload=AgentTurnResponseTurnCompletePayload(
-                            turn=turn,
-                        )
+            chunk = AgentTurnResponseStreamChunk(
```

Comment:
when will this return `turn_awaiting_input`?

Reply:
shoot I'm deleting the wrong block..

Reply:
done